anaUltraScurve.py

```diff
@@ -249,9 +310,9 @@ def overlay_fit(VFAT, CHAN):
             noise[0] = param1
             fitENC.append(vToQm*param1*options.ztrim)
             pedestal[0] = param2
-            if noise[0] > 20.0 or ped_eff[0] > 50.0: mask[0] = True
-            else: mask[0] = False
-            masks[vfat].append(mask[0])
+            maskOutlier[0] = masks[vfat][chan]
+            maskDead[0] = int(masksDead[vfat][ch])
+            mask[0] = masks[vfat][chan]
```

Comment:
See comments above, I think we should rethink the implementation to be more descriptive to the user